lib/oli/groups.ex

```diff
@@ -4,9 +4,12 @@ defmodule Oli.Groups do
   """
 
   import Ecto.Query, warn: false
-  alias Oli.Repo
 
+  alias Oli.Accounts
+  alias Oli.Accounts.Author
   alias Oli.Groups.Community
+  alias Oli.Groups.CommunityAccount
```

Comment:
Update alias to `Oli.Groups.{Community, CommunityAccount}` - same thing w Accounts, Repo